tests/template_test.go

```diff
@@ -154,12 +152,12 @@ var _ = Describe("[Serial][sig-compute]Templates", Serial, decorators.SigCompute
 				for param, value := range templateParams {
 					ocProcessCommand = append(ocProcessCommand, "-p", fmt.Sprintf("%s=%s", param, value))
```

Comment:
Are these still running? IIRC we wanted to drop `oc` specific tests?

Reply:
Actually, IDK about this but if you agree I would leave this for a follow-up PR.
This PR already has a huge context and I prefer not to increase it more. WDYT?
@xpivarc Are you aware of the removal of `oc` stuff?

Reply:
We don't have a lane for it, so yes it would be reasonable to remove.

Reply:
:+1: Will check in a follow up PR. Thank you